jekyll/_cci2/authentication.md

```diff
@@ -0,0 +1,40 @@
+---
+ layout: classic-docs
+ title: "Authentication"
+ short-title: "Authentication"
+ description: "Configuring LDAP Authentication"
+ categories: [administration]
+ hide: true
+ order: 9
+---
+
+This document describes how to enable, configure, and test CircleCI to authenticate users with OpenLDAP or Active Directory credentials.
+
+## Prerequisites
+
+- Install and configure an LDAP instance and populate the database with your users and credentials.
```

Comment:
I would probably not recommend calling LDAP a database, it might be a bit confusing. Since setting up LDAP by itself is out of scope we can just say what prerequisite is having up and running LDAP server/AD

Reply:
thanks, I changed to "Install and configure your LDAP server and Active Directory."